packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -200,10 +201,10 @@ export class FileTreeModel extends CompressedTreeModel implements LocationServic
 
     protected async shouldReplace(fileName: string): Promise<boolean> {
         const dialog = new ConfirmDialog({
-            title: 'Replace file',
-            msg: `File '${fileName}' already exists in the destination folder. Do you want to replace it?`,
-            ok: 'Yes',
-            cancel: 'No'
+            title: nls.localize('theia/filesystem/replaceTitle', 'Replace file'),
```

Comment:
minor: I think the following would be a better title overall

```suggestion
            title: nls.localize('theia/filesystem/replaceTitle', 'Replace File'),
```